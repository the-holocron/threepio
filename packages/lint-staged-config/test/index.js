const config = require("../lintstagedrc.json");

describe("configuration", () => {
	describe("lint Staged", () => {
		test("should render the configuration", () => expect(config).toMatchSnapshot());
	});
});
