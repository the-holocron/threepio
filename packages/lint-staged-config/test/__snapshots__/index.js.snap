// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`configuration lint Staged should render the configuration 1`] = `
Object {
  "*.css": "stylelint --fix",
  "*.js": "eslint --fix",
  "*.scss": "stylelint --syntax=scss --fix",
  "*.{png,jpeg,jpg,gif,svg}": "imagemin-lint-staged",
}
`;
